src/courseware/course/sidebar/sidebars/course-outline/Section.jsx

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Button, Icon } from '@openedx/paragon';
+import {
+  CheckCircle as CheckCircleIcon,
+  ChevronRight as ChevronRightIcon,
+} from '@openedx/paragon/icons';
+
+import courseOutlineMessages from '../../../../../course-home/outline-tab/messages';
+import { CompletionSolidIcon } from './icons';
+
+const Section = ({ intl, section, handleSelectSection }) => {
+  const {
+    id,
+    complete,
+    title,
+  } = section;
+
+  const sectionTitle = (
+    <>
+      <div className="col-auto p-0" style={{ fontSize: '1.1rem' }}>
```

Comment:
[question]: Can we move `style={{ fontSize: '1.1rem' }}` to a styles file?

Reply:
Removed, no longer used